nodeconfigs.json

```diff
@@ -319,6 +348,37 @@
         "gcc": "/opt/local/gcc49/bin/gcc",
         "build_tag": "zone"
     },
+    {
+        "// image": "minimal-64-lts 19.4.0",
+        "image": "5417ab20-3156-11ea-8b19-2b66f5e7a439",
+        "version": "v6.17.0",
```

Comment:
I guess if we wanted to consider bumping the node (latest are v6.17.1 and v8.17.0) and npm versions (latest is v6.13.6), we would do that in a separate release. Especially true for v6, because currently 31 components are using v6.17.0.

Reply:
Do we have a requirement to provide the same node version across releases? If not, then we certainly _could_ bump now, it just means that the bar to move to pkgsrc 19.4 is a bit higher then, and I'm not sure that's a good thing.

Reply:
Given that a build of, say, sdc-vmapi at a point in time will just use whatever the *latest* sdcnode build set is... if you remove the v6.17.0 build then the next sdc-vmapi build would break.

Ah... but this is for a new origin image (`NODE_PREBUILT_IMAGE`) then nothing would break. You'd only be requiring vmapi (or others) to update their node version at the same time as they considered updating their origin image.  That's a slightly higher burden on that updater, but not a biggie.

The ideal is that we'd add the new latest node version *and* have the currently used version(s). Then all components would update to the latest node version reasonably quickly.

So, I'd suggest either:
1. add both v6.17.0 and v6.17.1 (ditto for two v8 versions), or
2. add just the latest versions of v6 and v8

Reply:
Cool. I'll just add the latest versions then. I'll fire off a test build of sdcnode with this change once it's pushed to this dev branch.